CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/ImportJDBCIO.java

```diff
@@ -55,7 +55,7 @@
 
 public final class ImportJDBCIO {
 
-    private static final String IMPORT_DELIMITED_DIR = "ImportDelimited";
+    private static final String IMPORT_JDBC_DIR = "DatabaseImporterTemplates";
```

Comment:
Are you possibly able to change the name from "DatabaseImporterTemplates" to "ImportDatabase"? Just keeps it consistent with preferences and like saved elsewhere (being that the directory is just the name of the view)